config/schema/bento_sdk.schema.yml

```
@@ -84,4 +84,65 @@ bento_sdk.settings:
+            include_product_details:
+              type: boolean
+              label: 'Include Product Details'
+            include_customer_context:
+              type: boolean
+              label: 'Include Customer Context'
+            include_order_context:
+              type: boolean
+              label: 'Include Order Context'
+            include_product_images:
+              type: boolean
+              label: 'Include Product Images'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)